bin/dev/install_tools.sh

```diff
@@ -1,90 +1,49 @@
 #!/bin/bash
 set -o errexit -o nounset
 
-# Get version information
-. "$(dirname "$0")/versions.sh"
+# Get version information and set destination paths
+. "$(dirname "$0")/../common/versions.sh"
 
 # Tool locations
 s3="https://cf-opensusefs2.s3.amazonaws.com/fissile"
 
 # Tool versions
 thefissile="fissile-$(echo "${FISSILE_VERSION}" | sed -e 's/+/%2B/')"
 
-# Installs tools needed to build and run HCF
-bin_dir="${bin_dir:-output/bin}"
-tools_dir="${tools_dir:-output/tools}"
-ubuntu_image="${ubuntu_image:-ubuntu:${UBUNTU_VERSION}}"
 fissile_url="${fissile_url:-${s3}/${thefissile}.linux-amd64.tgz}"
-cf_url="${cf_url:-https://cli.run.pivotal.io/stable?release=linux64-binary&version=${CFCLI_VERSION}&source=github-rel}"
 stampy_url="${stampy_url:-https://github.com/SUSE/stampy/releases/download/${STAMPY_MAJOR}/stampy-${STAMPY_VERSION}.linux-amd64.tgz}"
-kubectl_url="${kubectl_url:-https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl}"
-k_url="${k_url:-https://github.com/aarondl/kctl/releases/download/v${K_VERSION}/kctl-linux-amd64}"
-kk_url="${kk_url:-https://gist.githubusercontent.com/jandubois/40a5b3756cf4bcbed940e6156272c0af/raw/${KK_VERSION}/kk}"
-helm_url="${helm_url:-https://kubernetes-helm.storage.googleapis.com/helm-v${HELM_VERSION}-linux-amd64.tar.gz}"
 
-mkdir -p "${bin_dir}"
-mkdir -p "${tools_dir}"
+mkdir -p "${SCF_BIN_DIR}"
+mkdir -p "${SCF_TOOLS_DIR}"
 
-bin_dir="$(cd "${bin_dir}" && pwd)"
+SCF_BIN_DIR="$(cd "${SCF_BIN_DIR}" && pwd)"
 
-echo "Fetching cf CLI $cf_url ..."
-wget -q "$cf_url"        -O "${tools_dir}/cf.tgz"
 echo "Fetching fissile $fissile_url ..."
 wget -q "$fissile_url"   -O - | tar xz --to-stdout fissile > "${FISSILE_BINARY}"
-echo "Installed: $("${FISSILE_BINARY}" version)"
 
 echo "Fetching stampy $stampy_url ..."
-wget -q "$stampy_url"   -O - | tar xz -C "${bin_dir}" stampy
-
-echo "Unpacking cf CLI ..."
-tar -xzf "${tools_dir}/cf.tgz" -C "${bin_dir}"
-
-echo "Fetching kubectl ${kubectl_url} ..."
-wget -q "${kubectl_url}" -O "${bin_dir}/kubectl"
-wget -q "${k_url}" -O "${bin_dir}/k"
-wget -q "${kk_url}" -O "${bin_dir}/kk"
-
-echo "Fetching helm from ${helm_url} ..."
-wget -q "${helm_url}" -O - | tar xz -C "${bin_dir}" --strip-components=1 linux-amd64/helm
+wget -q "$stampy_url"   -O - | tar xz -C "${SCF_BIN_DIR}" stampy
```

Comment:
This is run privileged by vagrant up, so no perm issues (I think). It's fine if the binary is root-owned, as long as it's still executable by vagrant. For non-vagrant runs, SCF_BIN_DIR shouldn't be /usr/local/bin